src/main/resources/public/jsx/components/SurveyCollection.js

```diff
@@ -68,6 +68,20 @@ export class SurveyCollection extends React.Component {
 
     getNodeById = (id) => this.props.surveyQuestions.find(sq => sq.id === id);
 
+    getRulesById = (id) => {
+        const rules = this.props.surveyRules.filter(rule => (rule.questionId && rule.questionId === id)
+            || (rule.questions && rule.questions.includes(id))
+            || (rule.questionSetIds1 && (rule.questionSetIds1.includes(id) || rule.questionSetIds2.includes(id)))
+            || (rule.question1 && rule.question1 === id ||rule.question2 === id));
+        return rules.map((r) => {
+            return r.questionId ? r.regex? `${'Rule: ' + r.ruleType + ' | Question: ' + r.questionsText + ' | Regex: '+r.regex} ` : `${'Rule: '+r.ruleType + ' | Question: ' + r.questionsText + ' | Min: ' + r.min + ' | Max: ' + r.max} `
```

Comment:
Make sure there is a space before and after `+`

Is the space at the end of each statement needed after } and before the tick symbol